profiling/src/timeline.rs

```diff
@@ -241,12 +218,73 @@ pub unsafe fn timeline_startup() {
             ptr::addr_of_mut!(FRANKENPHP_HANDLE_REQUEST_HANDLER),
             Some(ddog_php_prof_frankenphp_handle_request),
         ),
+        zend::datadog_php_zif_handler::new(
+            CStr::from_bytes_with_nul_unchecked(b"stream_select\0"),
+            ptr::addr_of_mut!(STREAM_SELECT_HANDLER),
+            Some(ddog_php_prof_stream_select),
+        ),
+        zend::datadog_php_zif_handler::new(
+            CStr::from_bytes_with_nul_unchecked(b"socket_select\0"),
+            ptr::addr_of_mut!(SOCKET_SELECT_HANDLER),
+            Some(ddog_php_prof_socket_select),
+        ),
+        zend::datadog_php_zif_handler::new(
+            CStr::from_bytes_with_nul_unchecked(b"socket_accept\0"),
```

Comment:
Not sure if socket_accept fits here, but if it does, stream_socket_accept should also fit here.

Reply:
I'll remove it from here and add it to my notes. Even though a call to `socket_accept()` might block (depending if the socket is blocking) it does not fit in this list of `select()`-type functions